srv/salt/_modules/cephprocesses.py

```diff
@@ -223,9 +223,7 @@ def add(self, prc, role):
         Also do a special check for oA
```

Comment:
Can the above mention of oA also go away?

Reply:
right, I grepped for openattic and omitted oA :/

Reply:
:-)